webpack.config.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
         loader: "babel",
         query: {
           presets: ["react", "es2015", "stage-3"],
+          plugins: [`${__dirname}/transform-code-string`],
```

Comment:
Yeah that's right - I removed the `presets` key here so it's less confusing/doesn't duplicate the config.